api-get-object-attributes.go

```diff
@@ -0,0 +1,155 @@
+package minio
+
+import (
+	"context"
+	"encoding/xml"
+	"errors"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/minio/minio-go/v7/pkg/encrypt"
+	"github.com/minio/minio-go/v7/pkg/s3utils"
+)
+
+// ObjectAttributesOptions ...
+// https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObjectAttributes.html
+type ObjectAttributesOptions struct {
+	ObjectAttributes     string
```

Comment:
It should be obvious how to use this SDK without needing to read the API specs. I don't really see any point in the user having to bother with filling this. 

We could accept a mask with named options - but I don't really see much point in filtering.